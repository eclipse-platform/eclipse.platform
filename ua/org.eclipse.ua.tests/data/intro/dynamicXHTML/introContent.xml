<?xml version="1.0" encoding="utf-8" ?>
<!--
     Copyright (c) 2005, 2006 IBM Corporation and others.

     This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     https://www.eclipse.org/legal/epl-2.0/

     SPDX-License-Identifier: EPL-2.0
    
     Contributors:
         IBM Corporation - initial API and implementation
 -->

<!--
   This is an intro used to test dynamic content with XHTML files. This lists all the pages, and
   the referenced XHTML files contain the content. See the files for more details.
-->
<introContent>
    <page id="root" content="content/root.xhtml"/>
    <page id="static" content="content/static.xhtml"/>
    <page id="filter" content="content/filter.xhtml"/>
    <page id="include" content="content/include.xhtml"/>
    <page id="include2" content="content/include2.xhtml"/>
    <page id="anchor" content="content/anchor.xhtml"/>
    <page id="anchor2" content="content/anchor2.xhtml"/>
</introContent>
