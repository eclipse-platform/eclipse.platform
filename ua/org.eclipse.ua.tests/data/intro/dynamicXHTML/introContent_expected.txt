Intro Model Content:
======================

Model has valid config = true
Presentation type = 2
Home page id = root
Number of pages (not including Root Page) = 8
Number of shared groups = 0
Number of unresolved extensions = 0

HOME PAGE: 
--------------
	is dynamic= true
	id = root
	title = null
	style = null
	alt-style = null
	url = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = static
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = filter
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = include
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = include2
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = anchor
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = anchor2
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = anchor2
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


PAGE id = anchor2
----------
	title = null
	style = null
	alt-style = null
	style-id = null
	page styles are = 
	page alt-styles are = 
	page children = 0


Model Flag Tests: 
----------------
		First page children are: 
			Groups: 0
			Links: 0
			Texts: 0
			HTMLs: 0
			Images: 0
			Includes: 0
			Page Titles: 0
			Page Heads: 0
			Model Elements: 0
			Containers: 0
			All Pages: 0
			Anchors: 0
			Content providers: 0
			Elements with Text child(AbstractTextElemets): 0
			Groups and Links: 0